.taskcluster.yml

```diff
@@ -149,13 +149,21 @@ tasks:
             git checkout {{event.head.sha}}
             source $HOME/servotidy-venv/bin/activate
             servo-tidy
+            sccache --stop-server || true
+            export RUST_LOG=sccache=debug
```

Comment:
Actually, after looking at [a build log on this PR](https://tools.taskcluster.net/groups/Ej71TayqSVufyFY8ENrgaA/tasks/Tdq8F9kqQ2yDgKbtHcuaHg/runs/0/logs/public%2Flogs%2Flive.log), you probably don't want to export this, just put it in the `--start-server` line. Otherwise you get log spew every time cargo invokes sccache.